zipkin-collector/rabbitmq/README.md

```diff
@@ -17,6 +17,7 @@ Property | Environment Variable | Description
 `zipkin.collector.rabbitmq.queue` | `RABBIT_QUEUE` | Queue from which to collect span messages. Defaults to `zipkin`
 `zipkin.collector.rabbitmq.username` | `RABBIT_USER` | Username to use when connecting to RabbitMQ. Defaults to `guest`
 `zipkin.collector.rabbitmq.virtualHost` | `RABBIT_VIRTUAL_HOST` | RabbitMQ virtual host to use. Defaults to `/`
+`zipkin.collector.rabbitmq.useSsl` | `RABBIT_USE_SSL` | Set to `true` to use SSL when connecting to RabbitMQ
```

Comment:
nit when documenting and testing properties, we use lower-hyphen (although seems not the case in this README)

Ex from zipkin-server-shared.yml
```
      # Using ssl for connection, rely on Keystore
      use-ssl: ${CASSANDRA_USE_SSL:false}
```

Reply:
Fixed